editor/assets/primitives.fbx.meta

```diff
@@ -74,7 +74,7 @@
       "subMetas": {},
       "userData": {
         "gltfIndex": 2,
-        "triangleCount": 72
+        "triangleCount": 92
```

Comment:
This is editor 3.8.6 change it. I think we should commit this meta file, otherwise, it's always dirty in git workspace.